JSTests/stress/uint8array-toBase64.js

```diff
@@ -0,0 +1,126 @@
+//@ requireOptions("--useUint8ArrayBase64Methods=1")
+
+function shouldBe(actual, expected) {
+    if (actual !== expected)
+        throw new Error(`FAIL: expected '${expected}' actual '${actual}'`);
+}
+
+shouldBe((new Uint8Array([])).toBase64(), "");
+shouldBe((new Uint8Array([0])).toBase64(), "AA==");
+shouldBe((new Uint8Array([1])).toBase64(), "AQ==");
+shouldBe((new Uint8Array([128])).toBase64(), "gA==");
+shouldBe((new Uint8Array([254])).toBase64(), "/g==");
+shouldBe((new Uint8Array([255])).toBase64(), "/w==");
+shouldBe((new Uint8Array([0, 1])).toBase64(), "AAE=");
+shouldBe((new Uint8Array([254, 255])).toBase64(), "/v8=");
+shouldBe((new Uint8Array([0, 1, 128, 254, 255])).toBase64(), "AAGA/v8=");
+
+shouldBe((new Uint8Array([])).toBase64({}), "");
+shouldBe((new Uint8Array([0])).toBase64({}), "AA==");
+shouldBe((new Uint8Array([1])).toBase64({}), "AQ==");
+shouldBe((new Uint8Array([128])).toBase64({}), "gA==");
+shouldBe((new Uint8Array([254])).toBase64({}), "/g==");
+shouldBe((new Uint8Array([255])).toBase64({}), "/w==");
+shouldBe((new Uint8Array([0, 1])).toBase64({}), "AAE=");
+shouldBe((new Uint8Array([254, 255])).toBase64({}), "/v8=");
+shouldBe((new Uint8Array([0, 1, 128, 254, 255])).toBase64({}), "AAGA/v8=");
+
+for (let alphabet of [undefined, "base64"]) {
+    shouldBe((new Uint8Array([])).toBase64({alphabet}), "");
+    shouldBe((new Uint8Array([0])).toBase64({alphabet}), "AA==");
+    shouldBe((new Uint8Array([1])).toBase64({alphabet}), "AQ==");
+    shouldBe((new Uint8Array([128])).toBase64({alphabet}), "gA==");
+    shouldBe((new Uint8Array([254])).toBase64({alphabet}), "/g==");
+    shouldBe((new Uint8Array([255])).toBase64({alphabet}), "/w==");
+    shouldBe((new Uint8Array([0, 1])).toBase64({alphabet}), "AAE=");
+    shouldBe((new Uint8Array([254, 255])).toBase64({alphabet}), "/v8=");
+    shouldBe((new Uint8Array([0, 1, 128, 254, 255])).toBase64({alphabet}), "AAGA/v8=");
+
+    shouldBe((new Uint8Array([])).toBase64({get alphabet() { return alphabet; }}), "");
+    shouldBe((new Uint8Array([0])).toBase64({get alphabet() { return alphabet; }}), "AA==");
+    shouldBe((new Uint8Array([1])).toBase64({get alphabet() { return alphabet; }}), "AQ==");
+    shouldBe((new Uint8Array([128])).toBase64({get alphabet() { return alphabet; }}), "gA==");
+    shouldBe((new Uint8Array([254])).toBase64({get alphabet() { return alphabet; }}), "/g==");
+    shouldBe((new Uint8Array([255])).toBase64({get alphabet() { return alphabet; }}), "/w==");
+    shouldBe((new Uint8Array([0, 1])).toBase64({get alphabet() { return alphabet; }}), "AAE=");
+    shouldBe((new Uint8Array([254, 255])).toBase64({get alphabet() { return alphabet; }}), "/v8=");
+    shouldBe((new Uint8Array([0, 1, 128, 254, 255])).toBase64({get alphabet() { return alphabet; }}), "AAGA/v8=");
+}
+
+shouldBe((new Uint8Array([])).toBase64({alphabet: "base64url"}), "");
+shouldBe((new Uint8Array([0])).toBase64({alphabet: "base64url"}), "AA");
```

Comment:
i just realized these are all missing padding `=`, which im fixing as a prerequisite in <https://github.com/WebKit/WebKit/pull/29530> (which should also help if/when `omitPadding` is approved in TC39 plenary later this month)